### PR TITLE
feat(#575): jsonpath support

### PR DESCRIPTION
**Pull request by pkarw**
Related Feature Request: #575 

I've added an easy way to support the [JsonPath](https://www.npmjs.com/package/jsonpath) which works great along with the built-in `fetch` resource.

Example usage:

```js

ask {

let cities = [
  { name: "London", "population": 8615246 },
  { name: "Berlin", "population": 3517424 },
  { name: "Madrid", "population": 3165235 },
  { name: "Rome",   "population": 2870528 }
];

let cityNames = cities:jsonPath('$..name');
// [ "London", "Berlin", "Madrid", "Rome" ]

}

```

### CODE REVIEW

**Review by czerwinskilukasz1 — CHANGES_REQUESTED** [review]
Piotr, congrats for your first(?) PR in AskQL! 😃 
JsonPath is a great feature.  One thing I would like to add is an AskScript test for JsonPath. While technically it's not required to have a test in AskScript to test the resource, the tests can easily serve as examples of usage.  I propose adding a folder `15-more-resources/` in `src/askscript/__tests__` and putting there the example you wrote in the description along with the expected value in `.test.result.ts` file.

**Comment by pkarw** [comment]
@czerwinskilukasz1 fixed :)

File: package.json [hunk 1]
```diff
@@ -59,7 +59,9 @@
   "engines": {
     "node": ">=12.14.0"
   },
-  "dependencies": {},
+  "dependencies": {
+    "jsonpath": "^1.1.1"
```
Comment (mhagmajer) [inline package.json:63]: We keep all dependencies as dev dependencies and people who wish to use 'jsonPath' would need to install it separately like they do with 'node-fetch'

Comment (pkarw) [reply package.json:63]: @mhagmajer fixed, thanks :)

**Comment by pkarw** [comment]
Conflicts solved. @mhagmajer / @czerwinskilukasz1 can you guys please merge it in?

File: src/askscript/__tests__/15-more-resources/more-resources-01-jsonPath.ask [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    let revPerClient = {
+        A: 136,
+        B: 426,
+        C: 133,
+        D: 35,
+        E: 246,
+        F: 446,
+        G: 53,
+    }
+    revPerClient:jsonPath('$.A')
```
Comment (mhagmajer) [inline src/askscript/__tests__/15-more-resources/more-resources-01-jsonPath.ask:11]: Would be great to provide more examples and return them as dictionary, thoughts?

Comment (pkarw) [reply src/askscript/__tests__/15-more-resources/more-resources-01-jsonPath.ask:11]: Done!

**Review by lukasz-eond — APPROVED** [review]
Thanks for adding the .ask file :) 
No more requests from me.

**Review by czerwinskilukasz1 — APPROVED** [review]
Thanks for adding the .ask file :) 
No more requests from me.

**Pull request merged**